app/uk/gov/hmrc/cdsimportsddsfrontend/services/DeclarationXml.scala

```diff
@@ -230,6 +245,10 @@ object DeclarationXml {
 
   }
 
+  private def maybeElement(name: String, maybeValue: Option[String]) = {
```

Comment:
what does this return if maybeValue is None?

Reply:
The actual return type is Any. If it's None or Some("") we don't want to return an element with no content or "", even though this is valid xml for optional elements